### DISCUSSION TITLE
install/import

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2356] [Announcements]
# install/import

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/import.html

**Comment by liyuerich** [comment]
同时指定导入多中离线资源

是不是“多种” 离线资源，错别字？

**Reply by windsonsea** [reply]
Seems this typos has been fixed, thanks for your carefulness.
